huxley/www/js/components/RegistrationView.js

```diff
@@ -112,301 +116,187 @@ var RegistrationView = React.createClass({
 
   render: function() {
     var conference = this.context.conference;
+
+    //I feel like this (And a lot of others in this section) could 
+    //Be reduced with some kind of mapping,
+    //But there's no obvious way to map over Objects in JS so :/
+    var accountInformationErrors = {
```

Comment:
Constructing objects was really just to make the code clearer to read and organization slightly more manageable. But I agree that it has its setbacks and if there's a better way to do it, I'm here to learn.

And I figure if we add/subtract anything to any of the subcomponents in the future, it's slightly easier to modify when we pass them as props to subcomponents rather than to pass each individual piece of data as it's own prop. That's just my instinct though.